server/web/routes/proposals.js

```diff
@@ -322,6 +323,51 @@ async function getObjectFromS3(fileName) {
   });  
 }
 
+async function findAllProposalsInSameGroup(proposal, includeSelf = false) {
```

Comment:
Remove the second parameter includeSelf, since the value of it, is always false. This just added unnecessary complication.